libs/ml/src/dataloaders/tensor_dataloader.cpp

```diff
@@ -79,6 +100,27 @@ bool TensorDataLoader<TensorType>::AddData(std::vector<TensorType> const &data,
   return true;
 }
 
+template <typename TensorType>
+OperationsCount TensorDataLoader<TensorType>::ChargeAddData(const std::vector<TensorType> &data,
+                                                            const TensorType &             labels)
+{
+  OperationsCount cost = 2;  // setting one_sample_label_shape
+  cost += 4;                 // resizing data vector
+  OperationsCount label_copy_cost = TensorType::PaddedSizeFromShape(labels.shape());
+  cost += label_copy_cost;
```

Comment:
seems like these should be one line + add a comment to be consistent with the prior two lines:
cost += PaddedSizeFromShape() // label copy cost